static/js/components/SourceDesktop.jsx

```diff
@@ -407,7 +411,7 @@ const SourceDesktop = ({ source }) => {
               <div className={classes.photometryContainer}>
                 <Suspense fallback={<div>Loading spectroscopy plot...</div>}>
                   <Plot
-                    url={`/api/internal/plot/spectroscopy/${source.id}?width=800&height=600`}
+                    url={`/api/internal/plot/spectroscopy/${source.id}?width=800&height=600&spectrumID=${specIDs}`}
```

Comment:
I think this approach can be simplified to the following:

```
<Plot ... cacheIDs=${specIDs}>
```

All we need is for a bunch of string identifiers to be passed to the plot that will change whenever the spectra change.  This way, there is also no need to parse the spectra IDs out---we can use their labels as-is.